### PR TITLE
Update Figma-broker documentation

### PR DESCRIPTION
**Pull request by vnys**
Missing information about updating images etc.

### CODE REVIEW

File: apps/figma-broker/README.md [hunk 1]
```diff
@@ -69,3 +69,7 @@ export const rootFontSize = 16
 1. Start the application with `pnpm start` or `pnpm run develop` for development
 1. Run commands from Commands.httpie by clicking the lines prefixed with ⚡ (requires the httpie vs code extension and a locally installed version of httpie)
```
Comment (mimarz) [inline apps/figma-broker/README.md:70]: We do have this entry for "how to check for images" along with comments in the `Commands.httpie` file

**Review by mimarz — CHANGES_REQUESTED** [review]
LGTM on the script 😅. Do we need more than that?

File: apps/figma-broker/README.md (hunk 1 above)
Comment (vnys) [reply apps/figma-broker/README.md:70]: No probably not. Need a 👍 then.

File: apps/figma-broker/README.md [hunk 2]
```diff
@@ -69,3 +69,7 @@ export const rootFontSize = 16
...
 
+## TODO
+
+- Add how to check for new images
+
```
Comment (mimarz) [inline apps/figma-broker/README.md:75]: Not needed todo ?

**Review by mimarz — CHANGES_REQUESTED** [review]
remove todo section?

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**